abb_hardware_interface/include/abb_hardware_interface/abb_system_position_only.hpp

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#pragma once
+#ifndef ABB_HARDWARE_INTERFACE__ABB_SYSTEM_POSITION_ONLY_HPP__
```

Comment:
It's not a huge deal, but what's the reasoning to switch the include guard from `#pragma once`?
It's a bit simpler, and we use it in the [MoveIt style guide](https://moveit.ros.org/documentation/contributing/code/)

Reply:
I'd also argue that we should be consistent with include guard usage. If you are OK with using `#pragma once`, could you change this in the other header files as well?

Reply:
I'm Ok with `#pragma once`. Using an include guard is just my habit and a while ago I've read on StackOverflow, that `#pragma once` is prone to bugs. But nevertheless, I will change this :)